common/rpc/interceptor/namespace_validator.go

```diff
@@ -224,6 +229,16 @@ func (ni *NamespaceValidatorInterceptor) StateValidationIntercept(
 	return handler(ctx, req)
 }
 
+// while most(all?) workflow related requests must have namespace field
+// admin request at most should
```

Comment:
I don't think I follow the comment here; it sounds like admin requests _should_ have a namespace field set? Maybe the comment could be clarified.

Reply:
what I mean - for that request namespace field is not a MUST. If it is not a MUST validation should not fail

Reply:
I understand, but I don't see how "admin request at most should" expresses that.